File: jupyter2/problems/iteration/minimize_polynomial/jl/minimize_polynomial.jl
### begin my answer

### end my answer

function main()
    @assert abs(min_value(-1, 1, 1000) - 0.903266) < 1.0e-6
    @assert abs(min_value( 1, 2, 1000) - 1.928766) < 1.0e-6
    println("OK")
end

main()

File: jupyter2/problems/iteration/minimize_polynomial/rs/minimize_polynomial.rs
/** begin my answer */

/** end my answer */

fn main() {
    assert!((min_value(-1.0, 1.0, 1000) - 0.903266).abs() < 1.0e-6);
    assert!((min_value( 1.0, 2.0, 1000) - 1.928766).abs() < 1.0e-6);
    println!("OK");
}
